hazelcast/src/main/java/com/hazelcast/internal/metrics/MetricTarget.java

```diff
@@ -40,26 +40,28 @@
     DIAGNOSTICS,
     JET_JOB;
 
-    public static final List<MetricTarget> ALL_TARGETS = unmodifiableList(asList(values()));
+    private static final MetricTarget[] VALUES = values();
```

Comment:
nit: does this change help with avoiding litter on the stats collection path? `values()` calls were used only in one-time static initializers, so that's why I'm asking.

Reply:
No need actually, reverted it.